content/cloud/pricing-and-billing.md

```diff
@@ -0,0 +1,114 @@
+title = "Pricing Plans and Billing"
+template = "cloud_main"
+date = "2023-06-20T00:07:00Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/payment-and=billing.md"
+
+---
+- [Fermyon Cloud Pricing](#fermyon-cloud-pricing)
+- [General Information](#general-information)
+  - [Payment](#payment)
+  - [Refund](#refund)
+  - [Support](#support)
+  - [Quota](#quota) 
+- [Next Steps](#next-steps)
+
+## Fermyon Cloud Pricing
+Fermyon Cloud offers subscription plans to help you increase the limits of your Fermyon Cloud usage. This page covers general information about subscription plans and answers frequently asked questions around [payment](#payment), [refunds](#refund), [support](#support), and [quota](#quota).
+
+As described in detail on the [Fermyon Pricing page](fermyon.com/pricing), there are 3 different plans crafted to fit your workloads' requirements: Starter, Growth, and Enterprise. 
+
+The Starter Plan provides a broad range of Fermyon Cloud services for you to try out, free of charger, forever. If you find that you're hitting the upper limits of the Starer Plan, we recommend promoting your account to the Growth Plan, where you can enjoy the same services with a higher quota limit. Lastly, if the serives offered don't quite meet your needs in the Growth Plan, [reach out](mailto:sales@fermyon.com) to the Fermyon team and we'll work to create a customized Enterprise Plan that will best meet your requirements. 
+
+> Please note that Growth and Enterprise Plans are billed on a monthly reoccuring basis. Start Plan is a free subscription, without an expiration date. 
+
+## General Information
+
+### Payment
+
+**I just signed up for the Growth Plan, when should I expect my first charge?**
+
+You should receive your first charge immediately after signing up for the Growth Plan. You can expect your second charge on the monthly anniversary of your signup.
+
+**What payment methods for you accept?**
+
+Most. One of the main reasons we chose to work with industry leader Stripe is because of the huge scope of payment options they offer, starting with the normal credit / debit card options, but extending to ACH, SEPA, Apple Pay / Google Pay and many more including a larger variety of global options. For more details and a list to all Stripe payment options see this Stripe Resource. 
+
+**Can I pay every month manually?**
+
+No, at this time Fermyon Cloud only accepts reoccuring payments. If at any point you decide you no longer want to continue with your paid plan, you are eligable to cancel your subscription. 
+
+**What happens if there's an issue with my payment method during a reoccuring charge?**
+
+You will receive an email from Fermyon informing you that the payment method is invalid and requesting an updated, valid form of payment. If the payment method is not updated by the next billing date, your limits will be reverted to the Starter Subscription.
```

Comment:
```suggestion
Fermyon will send you an email informing you that the payment method is invalid and requesting an updated, valid form of payment. If the payment method is not updated by the next billing date, your limits will be reverted to the Starter Subscription.
```

(suggesting the change because we're not in control of them receiving it)

Also, do we only send one email?

Not quite sure about the tone of "that the payment method is invalid."  Okay, something went wrong in the payment.  That doesn't mean the method is "invalid."

Reply:
We (or rather Stripe) send a few e-emails. We continue to retry for a number of days, after which they are then downgraded to the Starter plan.

Stripe uses the term "declined" to cover payments that the issuer declines. So perhaps this?

"Fermyon will send you an email informing you that a recent payment has been declined and..."